services/src/main/java/org/jboss/windup/web/services/rest/WindupEndpointImpl.java

```diff
@@ -55,31 +55,62 @@
     private Queue executorQueue;
 
     @Override
+    public WindupExecution executeProjectWithContext(AnalysisContext analysisContext, Long projectId)
+    {
+        if (analysisContext == null)
+        {
+            analysisContext = this.analysisContextService.getDefaultProjectAnalysisContext(projectId);
+            if (analysisContext == null)
```

Comment:
So if there is no analysisContext provided in parameter, but there is a default context, you proceed to activating execution with that context? 
(Which leads into deleting that default context, since it will be used for execution)